modale.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Modale</title>
+</head>
+
+<style>
+    .modale{
+        display: flex;
+        flex-direction: column;
+        box-sizing: border-box;
+        padding:30px;
+        border: solid black;
+        /*width: 300px;
+        height: 100px;*/
+        position: absolute;
+        z-index: 2;
+        background-color: white;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+        border-radius: 10px;
```

Comment:
review how you positioned the modal, with these rules you moved it forth and back again.